packages/react-components/src/index.ts

```diff
@@ -18,5 +18,7 @@ export {
 export { useViewport } from './hooks/useViewport';
 export { TimeSync } from './components/time-sync';
 
+export { Kpi as UnstableKpi } from './components/kpi/kpi';
```

Comment:
I think we might need to export the styles as well? I think there is a css file for both of these.

Reply:
your correct, need to alter the build system and figure out how we want to build the styles hopefully in a way that doesn't require additional style imports